Automattic/jetpack
# Social | Improve Quick Share buttons UI

Quick share buttons in Jetpack/Social sidebar and the post publish panel had some a11y issues which this PR fixes and also updates the design for the share buttons:

## Proposed changes:

* Updates the design for the Quick Share buttons - XPSW2jfYPEVyIAOQOWXijm-fi-3942%3A1676
* Removes the modal and dropdown for the quick share panel
* Also improves the `IconTooltip` component which didn't work well on smaller screens

### Other information:


## Jetpack product discussion
PT: pdrWKz-1h5-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

- Goto Post edit page
- Create a post and hit publish
- On the post publish panel, confirm that you see the quick share buttons inline instead of a dropdown
- Confirm that "Learn more" link points to the Jetpack Social support page
- Close the post publish sidebar
- Open the Jetpack or Social sidebar
- Confirm that you see the quick share buttons under "Share this post" section
- Click on the info icon
- Confirm that you see the text as expected
- Confirm that "Learn more" link points to the Jetpack Social support page

Desktop

<img width="377" alt="Screenshot 2023-12-21 at 7 24 10 PM" src="https://github.com/Automattic/jetpack/assets/18226415/dba7995d-6f95-4a6a-ae2f-b6efaf4a3636">
<img width="467" alt="Screenshot 2023-12-21 at 7 24 01 PM" src="https://github.com/Automattic/jetpack/assets/18226415/3845f5c8-d034-497d-82fc-742c5ef4af0a">
<img width="331" alt="Screenshot 2023-12-21 at 7 26 36 PM" src="https://github.com/Automattic/jetpack/assets/18226415/8d4c5d52-f789-4a3a-a05c-afa4aa19c3c2">

Mobile
![IMG_5265](https://github.com/Automattic/jetpack/assets/18226415/419d2e55-79d4-4631-a7fe-3ef1ff3349c4)
